### ISSUE TITLE
when we can not resolve field, it passes null as $contextValue for resolve function

### ISSUE THREAD

**Issue by hartono-sulaiman-kaskus** [Youshido/GraphQL#28]
Hi

```
class PostType extends \Youshido\GraphQL\Type\Object\AbstractObjectType
{

    /**
     * @param \Youshido\GraphQL\Config\Object\ObjectTypeConfig|TypeConfigInterface $config
     * @return mixed
     */
    public function build($config)
    {
        $config->addField('id', [
            'type' => new IdType(),
            'resolve' => function ($value, $args){
                return $value['post_id'];
            },
        ]);

```

in my db, the id field is named `post_id`, i want to resolve `post_id` to `id`
but in resolve function, $value is always null

it seems you have issue in https://github.com/Youshido/GraphQL/blob/master/src/Execution/Processor.php#L274

because 

```
        if (!$resolved && $field->getType()->getNamedType()->getKind() == TypeMap::KIND_SCALAR) {
            $resolved = true;
        }
        if ($resolveFunction = $field->getConfig()->getResolveFunction()) {
           .....
            } else {
                $resolverValue = $resolveFunction($resolved ? $resolverValue : $contextValue, $fieldAst->getKeyValueArguments(), $resolveInfo);
            }
        }
```

when $resolved = true and $resolverValue = null, $resolveFunction will have null $contextValue

**Comment by viniychuk** [comment]
Checking.

**Comment by viniychuk** [comment]
Fixed in v1.1.5

I added a test for field_alias resolver here: https://github.com/Youshido/GraphQL/blob/master/Tests/Schema/ProcessorTest.php#L199
